src/test/java/org/fundacionjala/coding/escarleth/Kata4Test.java

```diff
@@ -0,0 +1,30 @@
+package org.fundacionjala.coding.escarleth;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * This class contains UnitTests for Kata4.
+ */
+public class Kata4Test {
+
+    /**
+     * This method is for testing a digitalRoot number.
+     */
+    @Test
+    public void testDigitalRoot() {
+        Kata4 kata4 = new Kata4();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AIndentationCheck)